src/plugins/intel_cpu/src/utils/shape_inference/shape_inference_pass_through.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "shape_inference_cpu.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+class ShapeInferPassThrough final : public ShapeInferEmptyPads {
```

Comment:
Would be nice to have brief comment for each custom ShapeInfer class that will describe its base behavior.
This should help developers to understand which type use in their particular case.

Reply:
All the necessary descriptions were added, except for the `EyeShapeInferFactory` since technically it should be removed.

Reply:
Thats really nice amd helpfull!
However initially I mentioned: NgraphShapeInfer, PassThroughShapeInferFactory and InternalDynShapeInfer. The descriptions should help developer to choose correct ShapeInfer type for their particular node,

Reply:
Sure. All the necessary comments were added.